Interpreters/ManualInterpreter/Properties/AssemblyInfo.cs

```diff
@@ -32,5 +26,3 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]

```

Comment:
All the assemblyinfo files can be deleted. Afaik, in .NET Core, the same metadata is provided through .csproj instead.

Reply:
Removed